addons/account/models/account_journal.py

```diff
@@ -167,6 +167,11 @@ def _default_invoice_reference_model(self):
         string='Loss Account',
         domain="[('deprecated', '=', False), \
                 ('account_type', '=', 'expense')]")
+    account_cash_rounding_id = fields.Many2one(
```

Comment:
Why is this defined on the journal and not on the company?
It could also be implemented (or not implemented) using `ir.default`

Reply:
@william-andre, we set default cash rounding on journal because it is possible that multiple journals can have different cash rounding method that's why it's defined on journal.

Reply:
A cash journal in Belgium will have a 0.05€ cash rounding, but a bank journal not. Similarly, an internet order B2C might have a different rouding than a sale order B2B.

Reply:
But this field is set in the Customer Invoice/Vendor Bill journal, not the Cash/Bank journal

I don't understand the link with internet order and sale order though.
A sale order will probably end up in the same journal as the Customer Invoice journal

And shouldn't the SO/... set the right `invoice_cash_rounding_id`? (like the PoS is doing for instance)